ground/src/main/java/com/google/android/ground/model/TestModelBuilders.java

```diff
@@ -17,9 +17,9 @@
 package com.google.android.ground.model;
 
 import com.google.android.ground.model.job.Job;
+import com.google.android.ground.model.locationofinterest.AreaOfInterest;
```

Comment:
Unrelated: I'd be in favor of using `loi` for package and variable names.

Reply:
sgtm

Reply:
Also, let's discuss how we can consolidate our representation of point, polygon, and geojson in code and in the db.